### ISSUE TITLE
Not sure what to do here

### ISSUE THREAD

**Issue by lewisl** [JuliaImages/ImageView.jl#157]
I tried to add ImageView and got:

```
(v0.7) pkg> add ImageView
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package ImageView [86fae568]:
 ImageView [86fae568] log:
 ├─possible versions are: [0.0.1-0.0.23, 0.1.0-0.1.20, 0.2.0, 0.3.0-0.3.2, 0.4.0, 0.5.0-0.5.1, 0.6.0, 0.7.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.0.1-0.0.23, 0.1.0-0.1.20, 0.2.0, 0.3.0-0.3.2, 0.4.0, 0.5.0-0.5.1, 0.6.0, 0.7.0]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```

Sorry, I can't parse this.  What requirement is not being met?    I am running Julia 0.7 until I get my own code fixed and packages are mostly settled.

Thanks.

**Comment by timholy** [comment]
There isn't a released version of ImageView that works on 0.7 (we deliberately made this fail with https://github.com/JuliaLang/METADATA.jl/pull/17488).

However, you're in luck, see #156. There's some build error that prevents it from merging & tagging (https://github.com/JuliaIO/ImageMagick.jl/issues/133), but you can manually check that branch out and see if it works for you.

**Comment by timholy** [comment]
Fixed on master, not yet tagged though.
